docs/apu_flashing_ipxe.md

```diff
@@ -103,10 +103,12 @@ OS booting should start.
 
 Number of OSes can be increased in the future.
 
-> For legacy versions of coreboot PXE server boot menu is incorrectly 
-displayed. Every displayed character is doubled.
+> When `Legacy Console Redirection` is turned on displayed characters are 
+doubled. It's beacause iPXE is outputting data to the serial console and
+to the standard output, which is serial console in the case described in this
```

Comment:
better to use `to the screen` instead of `standard output`. Then say that screen is emulated on serial console